runners/katacoda/dirUtils.ts

```diff
@@ -0,0 +1,51 @@
+import { captureRejectionSymbol } from "events";
+import * as path from 'path';
+
+export class DirUtils{
+
+    getCdParam(currentDir:string, dir:string):string{
+
```

Comment:
Better to name 'dir' to 'targetDir' (or something similar) to indicate that this is directory to navigate into.